code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -393,6 +393,7 @@
 	for(var/mob/dead/observer/O in src.loc)
 		if(!O.client)	continue
 		if(O.mind && O.mind.current && O.mind.current.stat != DEAD)	continue
+		if(!(jobban_isbanned(O, "catban") || jobban_isbanned(O, "cluwneban")))	continue
```

Comment:
Remove the ! so you have the following:

`if (jobban_isbanned(O, "catban") || jobban_isbanned(I, "cluwneban")) continue`

This means if they're catbanned OR they're cluwnebanned then jump straight to the next iteration of the loop (and  do not execute anything after)